semantic-source/src/Source/Range.hs

```diff
@@ -19,7 +20,7 @@ data Range = Range
   { start :: {-# UNPACK #-} !Int
   , end   :: {-# UNPACK #-} !Int
   }
-  deriving (Eq, Generic, Ord, Show)
+  deriving (Eq, Generic, Ord, Show, ToJSON)
```

Comment:
We should probably 🔥 this change for now since we’re using `()` in the executable instead.

Reply:
Wasn't the intention to eventually have `ToJSON` instances for `Range` and `Span` so we can replace `()` with that?

Reply:
It is, but I’m not sure we want to derive it, and if we’re going to be investigating it in detail in a separate PR, let’s not merge something potentially incomplete and/or inconsistent with the way existing JSON formatting stuff for ranges/spans in `semantic` works (e.g. the `ToJSONFields` instance(s)) in the meantime.